### PR TITLE
fix: [Examus-429] updates russian translations

### PR DESCRIPTION
**Pull request by YuriiBorovenskyi**
**Description:**

* updates / fixes russian translations;
* updates version of `edx-proctoring` plugin to 1.6.1.

**YouTrack:** https://youtrack.raccoongang.com/issue/Examus-429

**Related MR:** 
* [examus/edx-platform]()
* [examus/edx-examus-proctoring]()

**Merge checklist:**

* [ ] All reviewers approved
* [ ] CI build is green
* [x] Demo status: OK
* [ ] All related documentation is updated
* [ ] DevOps notes added

**Post-Merge:**
* [ ] Create `ironwood-v.1.6.1` tag matching the new version number. (Will be made after merge)

### CODE REVIEW

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -3,193 +3,300 @@
...
-msgid "Proctoring Session Results Update for {course_name} {exam_name}"
+msgid "Proctoring Results For {course_name} {exam_name}"
+msgstr "Результаты прокторинга за экзаменом «{exam_name}» курса «{course_name}»"
+
+#: api.py:1072
+#, python-brace-format
+msgid "Proctoring Review In Progress For {course_name} {exam_name}"
 msgstr ""
-"Обновление результатов сдачи наблюдаемых заданий «{exam_name}» курса "
+"Выполняется рассмотрение прокторинга за экзаменом «{exam_name}» курса "
```
Comment (arsentur) [inline edx_proctoring/locale/ru/LC_MESSAGES/django.po]: It is correct? `прокторинга за экзаменом`

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -3,193 +3,300 @@
...
+#: api.py:1444
 msgid "Taking As Open Exam"
-msgstr "Сдаётся без наблюдения"
+msgstr "Сдаётся без прокторинга"
 
-#: api.py:1299 api.py:1304
+#: api.py:1449 api.py:1454
 msgid "Pending Session Review"
-msgstr "Ожидание отсмотра"
+msgstr "Ожидание рассмотрения сеанса"
```
Comment (arsentur) [inline edx_proctoring/locale/ru/LC_MESSAGES/django.po]: проверки?

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -3,193 +3,300 @@
...
 "\n"
+"        Your proctored exam \"%(exam_name)s\" in\n"
+"        <a href=\"%(course_url)s\">%(course_name)s</a> was reviewed and you\n"
+"        met all exam requirements.\n"
+"    "
+msgstr ""
+"\n"
+"Ваш экзамен с прокторингом \"%(exam_name)s\" \n"
+"<a href=\"%(course_url)s\">%(course_name)s</a> был рассмотрен и вы\n"
+"выполнили все требования прокторинга за экзаменом.\n"
```
Comment (arsentur) [inline edx_proctoring/locale/ru/LC_MESSAGES/django.po]: `за экзаменом`

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -3,193 +3,300 @@
...
+"        rules were followed. You should receive an email with your updated "
+"exam\n"
+"        status within 5 business days.\n"
+"    "
+msgstr ""
+"\n"
+"Ваш экзамен с прокторингом \"%(exam_name)s\"\n"
+"<a href=\"%(course_url)s\">%(course_name)s</a> был успешно отправлен\n"
+"и теперь будет рассмотрен на предмет соблюдения всех правил прокторинга\n"
+"за экзаменом. Вы получите электронное письмо с обновленным статусом экзамена "
```
Comment (arsentur) [inline edx_proctoring/locale/ru/LC_MESSAGES/django.po]: ditto

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -3,193 +3,300 @@
...
+"        exam requirements.\n"
+"    "
+msgstr ""
 "\n"
+"Ваш экзамен с прокторингом \"%(exam_name)s\" на курсе\n"
+"<a href=\"%(course_url)s\">%(course_name)s</a> был рассмотрен, и команда\n"
+"обнаружила одно или несколько нарушений правил прокторинга за экзаменом. "
+"Примеры\n"
+"поведения, которое может привести к нарушению правил, включают просмотр\n"
+"Интернета, использование телефона или получение помощи от другого человека. "
```
Comment (arsentur) [inline edx_proctoring/locale/ru/LC_MESSAGES/django.po]: Просмотр Интернета?

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -3,193 +3,300 @@
...
+"В \n"
+"результате нарушения (ий) вы не выполнили требования прокторинга "
+"за экзаменом.\n"
```
Comment (arsentur) [inline edx_proctoring/locale/ru/LC_MESSAGES/django.po]: за экзаменом.

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -198,18 +305,21 @@ msgid ""
...
 "\n"
-"Попробовать сдать наблюдаемый экзамен\n"
+"Попробовать сдать экзамен с прокторингом\n"
 "    "
 
 #: templates/practice_exam/entrance.html:9
 msgid ""
 "\n"
-"      Get familiar with proctoring for real exams later in the course. This practice exam has no impact\n"
+"      Get familiar with proctoring for real exams later in the course. This "
```
Comment (arsentur) [inline edx_proctoring/locale/ru/LC_MESSAGES/django.po:314]: I suppose that these changes were made automatically.

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po (hunk 1 above)
Comment (YuriiBorovenskyi) [reply edx_proctoring/locale/ru/LC_MESSAGES/django.po]: Прокторинг – это процедура наблюдения и контроля за дистанционным испытанием.
It is a definition from Internet.
Actually i replaced from `прокторинг за экзаменом` to `прокторинг экзамена`  everywhere.

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po (hunk 2 above)
Comment (YuriiBorovenskyi) [reply edx_proctoring/locale/ru/LC_MESSAGES/django.po]: Agree)

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po (hunk 3 above)
Comment (YuriiBorovenskyi) [reply edx_proctoring/locale/ru/LC_MESSAGES/django.po]: fixed

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po (hunk 4 above)
Comment (YuriiBorovenskyi) [reply edx_proctoring/locale/ru/LC_MESSAGES/django.po]: fixed

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po (hunk 5 above)
Comment (YuriiBorovenskyi) [reply edx_proctoring/locale/ru/LC_MESSAGES/django.po]: использование сторонних веб-сайтов

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po (hunk 6 above)
Comment (YuriiBorovenskyi) [reply edx_proctoring/locale/ru/LC_MESSAGES/django.po]: fixed

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po (hunk 7 above)
Comment (YuriiBorovenskyi) [reply edx_proctoring/locale/ru/LC_MESSAGES/django.po:314]: You are right.

File: edx_proctoring/locale/ru/LC_MESSAGES/django.po (hunk 1 above)
Comment (arsentur) [reply edx_proctoring/locale/ru/LC_MESSAGES/django.po]: I asked about `за`.

**Review by arsentur — APPROVED** [review]

**Pull request merged**